docs/fast_data/configuration/single_view_creator.md

```diff
@@ -103,150 +103,140 @@ service.addHook('onClose', async() => {
+#### Replace vs Update
 
-```js
-// (logger: BasicLogger, singleView: Document) => Boolean
-function singleViewValidator(logger, singleView) {
-  ... checks on singleView
+While the replace strategy covers every basic case, the update strategy provides a way to divide the aggregation effort between more than one Single View Creator.
+For example, we have a Single View called posts with a relationship to comments:
 
-  // returns a boolean
-  return validationResult
+```typescript [title="posts"]
+{
+  title: string,
+  comments: {text: string}[]
 }
 ```
 
-:::warning
-When the update of an existing Single View is triggered and the validation has a negative outcome, the Single View won't be updated, and instead it will be deleted.
-:::
-
-### Plugin
-
-Since version `v3.5.0`, it is possible to specify a custom validator function inside the configuration folder (`CONFIGURATION_FOLDER`).
-
-The file must be named `validator.js` and must export a function that will take as arguments the same as the default validator explained above.
+And then we have two Single View Creators, each one configured with the `update` strategy to update only one of the fields.
+The `aggregation.json` files would look like that:
 
-```js title="validator.js"
-module.exports = function validator(logger, singleView) {
-  ... custom validation logic on singleView
+```json [title="svc-1-aggregation.json"]
+{
+  "version": "1.3.0",
+   "config": {
+      "SV_CONFIG": {
+         "dependencies": {
+            "posts": {
+               "type": "projection",
+               "on": "_identifier",
+            },
+         },
+         "mapping": {
+            "title":"posts.title"
+         }
+      }
+   }
+}
+```
 
-  // returns a boolean
-  return customValidationResult
+```json [title="svc-2-aggregation.json"]
+{
+   "version": "1.3.0",
+   "config": {
+      "SV_CONFIG": {
+         "dependencies": {
+            "posts": {
+               "type": "projection",
+               "on": "_identifier"
+            },
+            "COMMENTS": {
+               "type": "config"
+            }
+         },
+         "mapping": {
+            "comments": "COMMENTS"
+         }
+      },
+      "COMMENTS": {
+         "dependencies": {
+            "comments": {
+               "on": "posts_to_comments",
+               "type": "projection"
+            }
+         },
+         "joinDependency": "comments",
+         "mapping": {
+            "text": "comments.text"
+         }
+      }
+   }
 }
 ```
 
-### Template
+Now we only need to configure one of the Single View Creators to react to changes on the `posts` projection and the other one to the `comments` projection. This is specially useful when one of the computation efforts is much bigger than the other so it makes sense to dedicate a Single View Creator with more resources to deal with the workload and not block the queue for what could be faster aggregations.
```

Comment:
In general, it makes sense. Only a couple of things:
1. for the two json, let's add a caption or something that says aggregation.json for svc-1 and aggregation.json for svc-2
2. after the example explained, I would just briefly recap how you would configure the two SVCs: just quickly recapping like svc-1 will have aggregation1.json and the UPSERT_STRATEGY set to upsert and so on

sounds reasonable?

Reply:
done!